lighthouse-core/test/gather/driver-test.js

```diff
@@ -166,6 +166,7 @@ describe('Browser Driver', () => {
       assert.ok(false, 'long-running getRequestContent supposed to reject');
     }, e => {
       assert.equal(e.code, 'PROTOCOL_TIMEOUT');
+      expect(e.friendlyMessage).toBeDisplayString(/^Waiting for DevTools.*Method: .+/);
```

Comment:
it is possible to get a test method in here so we can make sure the substitution is working?

Reply:
The .+ is 1 or more characters, so it will fail if the output is "...Method: ".  I wasn't sure if it would be consistently calling: `Method: Network.getResponseBody`.  I suspected that was 100% called first time everytime, but was just playing it safe.

Reply:
right, I think it's probably worth asserting. It's a bit more roundabout than the other tests that are getting touched in here, but the connection override intentionally makes `Network.getResponseBody` take longer than the `MAX_WAIT_FOR_PROTOCOL`, so it should always throw on that method.

Since we're already breaking into the function implementation for the test (overriding a specific driver method to make sure it will timeout), it seems ok to add to that so we can check the argument substitution as well.